Renderer/include/TwFonts.h

```diff
@@ -60,7 +60,7 @@ extern CTexFont *g_DefaultLargeFont;
 extern CTexFont *g_DefaultFixed1Font;
 extern CTexFont *g_DefaultFixedRuFont;
 
-void TwGenerateDefaultFonts(float _Scaling=1.0f, bool _forD3D=false);
+void TwGenerateDefaultFonts(float _Scaling, bool _forD3D);
 void TwDeleteDefaultFonts();
```

Comment:
Any good reason to change the API?

You can install the Renderer as its own static lib and use it for your own GUI stuff. In that case, the defaults make sense. Although they might have been the reason why the D3D flag was forgotten in the original code.

Reply:
When the D3D flag is false the font is displaced in Directx11.